tic-tac-toe/src/app/screens/Login/layout.js

```diff
@@ -0,0 +1,39 @@
+import React from 'react';
+import { Field, reduxForm } from 'redux-form';
+import PropTypes from 'prop-types';
+
+import style from './styles.scss';
+import { required, minLength, email } from './validation';
+import customField from './components/Field';
+
+function LoginForm({ handleSubmit }) {
+  return (
+    <form onSubmit={handleSubmit} className={style.login}>
+      <Field
+        name="username"
```

Comment:
let's extract the name of the field to another file in the same folder call: fieldNames.js
We usually use a separate file like that.